crypto/ca.go

```diff
@@ -106,13 +106,18 @@ func GenerateServerCert(config_obj *config_proto.Config, name string) (*CertBund
 	start_time := time.Now()
 	end_time := start_time.Add(365 * 24 * time.Hour)
 
-	serial_number := big.NewInt(1)
+	serialNumberLimit := new(big.Int).Lsh(big.NewInt(1), 128)
+	serialNumber, err := rand.Int(rand.Reader, serialNumberLimit)
```

Comment:
Just as a BTW, rand.Int will make a 64 bit int which will never wrap on 128 bit boundary above. 

Only a minor point but the Mod() code below is a noop actually (you cant really wrap a 64 bit int on 128 bit boundary in a reasonable number of increments).

Reply:
It was mainly to fix the edge-case. But I fully agree that it most often is a no-op.